SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -1,50 +1,66 @@
 
-### Getting the Unreal Engine fork source code and Unreal Linux cross-platform support
-1. In a terminal window, clone the [Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) repository. (You may get a 404 from this link. See  the instructions above, under _Unreal Engine EULA_, on how to get access.) <br/>
-Check out the fork by running either:
-    * (HTTPS) `git clone https://github.com/improbableio/UnrealEngine.git -b 4.20-SpatialOSUnrealGDK`
-    * (SSH) `git clone git@github.com:improbableio/UnrealEngine.git -b 4.20-SpatialOSUnrealGDK`
-1. To build Unreal server-workers for SpatialOS deployments, you need to build targeting Linux. This requires cross-compilation of your SpatialOS project and the Unreal Engine fork.
+### Step 2: Getting the Unreal Engine fork source code and Unreal Linux cross-platform support
 
-    In Unreal's [Compiling for Linux](https://wiki.unrealengine.com/Compiling_For_Linux) documentation, in the **getting the toolchain** section, click v11 **clang 5.0.0-based** to download the archive **v11_clang-5.0.0-centos7.zip** containing the Linux cross-compilation toolchain, then unzip.
+> You should clone the Unreal Engine fork into your root directory to avoid file path length errors. <br/>For example: C:\GitHub\UnrealEngine
 
-### Adding environment variables
+1\.  Clone the [Unreal Engine fork](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK) repository by opening a terminal and running either of these commands: 
 
-You need to add two environment variables: one to set the path to the Unreal Engine fork directory, and another one to set the path to the Linux cross-platform support.
+> You may get a 404 from this link. See  the instructions above, under _Unreal Engine EULA_, on how to get access to this repository. 
+
+|     |     |
+| --- | --- |
+| HTTPS | `git clone https://github.com/improbableio/UnrealEngine.git -b 4.20-SpatialOSUnrealGDK` |
+| SSH |`git clone git@github.com:improbableio/UnrealEngine.git -b 4.20-SpatialOSUnrealGDK`
+
+2\.  To build Unreal server-workers for SpatialOS deployments you need to build the Unreal Engine fork targeting Linux. This requires cross-compilation of your SpatialOS project and the Unreal Engine fork.
+
+In Unreal's [Compiling for Linux](https://wiki.unrealengine.com/Compiling_For_Linux) documentation, in the **getting the toolchain** section, click **v11 clang 5.0.0-based** to download the **v11_clang-5.0.0-centos7.zip** archive, then unzip this file into a suitable directory.
+
+### Step 3: Adding environment variables
+
+You need to add two [environment variables](https://docs.microsoft.com/en-us/windows/desktop/shell/user-environment-variables): one to set the path to the Unreal Engine fork directory, and another one to set the path to the Linux cross-platform support directory.
```

Comment:
```suggestion
You need to add two [environment variables](https://docs.microsoft.com/en-us/windows/desktop/procthread/environment-variables): one to set the path to the Unreal Engine fork directory, and another one to set the path to the Linux cross-platform support directory.
```

Reply:
Changed the documentation this links to because, while I think the doc you chose is clearer and more concise, it only talks about user environment variables, and we're asking users to set system environment variables. This might cause confusion.

Reply:
Good catch. Think I just slammed the first link to environment variable documentation in there.